### DISCUSSION TITLE
GotoX/config/ActionFilter.ini 如何如何配置要使用 worker 的域名？？

### DISCUSSION THREAD

**Discussion by becapital** [SeaHOH/GotoX#188] [问答/咨询]
谢谢老师执教哈。
1,将完整的 worker 域名填写到 GotoX/config/Config.ini 的 [cfw/worker] 中，===这个我配置了
2,并在 GotoX/config/ActionFilter.ini 配置要使用 worker 的域名，即可正常使用。===这个不知道如何将worker域名添加在哪一行

**Comment by SeaHOH** [comment]
> 2,并在 GotoX/config/ActionFilter.ini 配置要使用 worker 的域名，即可正常使用。===这个不知道如何将worker域名添加在哪一行

忘记删除这段了，可以不用配置。

**Comment by becapital** [comment]
你好谢谢
PIP不能安装环境，求指教，老师
C:\Users\Da\Desktop\GotoX-master>pip install requirements.text
ERROR: Could not find a version that satisfies the requirement
requirements.text (from versions: none)
ERROR: No matching distribution found for requirements.text

C:\Users\Da\Desktop\GotoX-master>pip install requirements.txt
ERROR: Could not find a version that satisfies the requirement
requirements.txt (from versions: none)
ERROR: No matching distribution found for requirements.txt


SeaHOH ***@***.***> 于2021年6月21日周一 上午11:11写道：

> 2,并在 GotoX/config/ActionFilter.ini 配置要使用 worker
> 的域名，即可正常使用。===这个不知道如何将worker域名添加在哪一行
>
> 忘记删除这段了，可以不用配置。
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SeaHOH/GotoX/discussions/188#discussioncomment-895629>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGJ7MGKLGK5DQXBHZY6WIN3TT2UXTANCNFSM47AXBFZQ>
> .
>


-- 

Best Regards
Todd Zhao
DAYONG.ZHAO

**Reply by SeaHOH** [reply]
命令应该是 `pip install -r requirements.txt`， releases 页面也有提供打包的便携版，如果不熟悉 Python 用法的话可以使用它。
